haskell/haddock.bzl

```diff
@@ -85,6 +83,10 @@ def _haskell_doc_aspect_impl(target, ctx):
     ghc_args.add([x.path for x in set.to_list(target[HaskellLibraryInfo].source_files)])
     ghc_args.add(["-v0"])
 
+    # haddock flags should take precedence over ghc args, hence are in
+    # last position
+    ghc_args.add(hs.toolchain.haddock_flags)
```

Comment:
FYI, using `args.add` in order to append a list has been deprecated. This will result in an error in a future Bazel release. We should use `ghc_args.add_all() instead.

See https://docs.bazel.build/versions/master/skylark/backward-compatibility.html#new-args-api

Reply:
Thank you. Fixed.